client/post-editor/editor-confirmation-sidebar/index.jsx

```diff
@@ -144,46 +158,57 @@ class EditorConfirmationSidebar extends Component {
 		const isSidebarActive = this.props.status === 'open';
 		const isOverlayActive = this.props.status !== 'closed';
 
-		return (
-			<div
-				className={ classnames( {
-					'editor-confirmation-sidebar': true,
-					'is-active': isOverlayActive,
-				} ) }
-			>
-				{ this.renderPublishingBusyButton() }
+		if ( ! this.state.hasUnclosed ) {
+			return (
+				<div className="editor-confirmation-sidebar">
+					<div key="sidebar" className="editor-confirmation-sidebar__sidebar" />
+				</div>
+			);
+		}
 
+		return (
+			this.state.hasUnclosed && (
 				<div
 					className={ classnames( {
-						'editor-confirmation-sidebar__sidebar': true,
-						'is-active': isSidebarActive,
+						'editor-confirmation-sidebar': true,
+						'is-active': isOverlayActive,
```

Comment:
Opportunity for a little code cleanup: for always-on classes, you can write:
```js
classnames( 'editor-confirmation-sidebar', { 'is-active': isOverlayActive } );
```

Reply:
Oh, I didn't know that. Thanks!